holopy/core/io/vis.py

```diff
@@ -72,6 +72,48 @@ def show(o, scaling='auto', vert_axis='x', horiz_axis='y',
     else:
         raise VisualizationNotImplemented(o)
 
+def save(o, names, scaling='auto', vert_axis='x', horiz_axis='y',
+                    depth_axis='z', colour_axis='illumination'):
+    """
+    Saves a hologram or reconstruction to (a) file(s).
+
+    Parameters
+    ----------
+    o : xarray.DataArray or ndarray
+        Object to visualize.
+    names : str
+        Name(s) of the file(s). If there is only one image contained (e.g.
+        hologram), the name will be used as a file name. If o contains more
+        plottable images (e.g. reconstruction volume), this string will be
+        formatted to get a counting index inserted. Example:
+        "reconstruction-%i.png"
```

Comment:
As I said regarding #323, I personally think a list-like of filenames is better than a format string.

Reply:
Made it!